reports/overall.py

```diff
@@ -38,3 +39,4 @@ def print_report_per_date(overall_report):
         logger.info(overall_data['DataByCountry'])
         logger.info("|- Investment by Loan Originator")
         logger.info(overall_data['DataByLoanOriginator'])
+        logger.info(f"|- Estimated monthly passive income: {overall_data['EstimatedMonthlyIncome']:.2f}€")
```

Comment:
I've added this into the overall report and the calculation in the calculator. However at least for the information report most probably should go to a section called forecast or something along these lines.